test/jdk/com/sun/java/swing/plaf/gtk/TestFileChooserSingleDirectorySelection.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
```suggestion
 * Copyright (c) 2022, 2023, Oracle and/or its affiliates. All rights reserved.
```
You always preserve the first year.